src/silly_sprites/opengl/error.cpp

```diff
@@ -15,4 +18,8 @@ namespace sly::gl {
     [[nodiscard]] char const* GlError::what() const noexcept {
         return m_message.c_str();
     }
+
+    void error_callback(int code, char const* message) {
+        spdlog::critical("GLFW error ({}): {}", code, message);
+    }
 } // namespace sly::gl
```

Comment:
is this only for glfw?
should we call it something different?
`error_callback` sounds verry general.
or we make it configurable somehow that it can display more errors than glfw errors.

Reply:
renamed to `glfw_error_callback`